rapids/README.md

```diff
@@ -129,24 +131,26 @@ To use XGBoost4j with Spark 2
     *   Multi-node clusters with homogenous GPU configuration
 *   Software Requirements
     *   NVIDIA driver 410.48+
-    *   CUDA v10.2/v10.1/v10.0/v9.2
-    *   NCCL 2.4.7 and later
-*   `EXCLUSIVE_PROCESS` must be set for all GPUs in each NodeManager (set by default in this
-    initialization action)
-*   `spark.dynamicAllocation.enabled` property must be set to `false` for Spark (set by default 
-    in this initialization action)
+    *   CUDA 9.2+
```

Comment:
This was tested with the previously listed versions, not yet tested with CUDA 11.  
```suggestion
    *   CUDA v10.2/v10.1/v10.0/v9.2
```

Reply:
Makes sense, reverted.